Create a new class represent creature card in MTGA (#15)
drvinceknight/ertai: README.md

```

### How to create a ceature card

> This can only create a creature with **power/toughness** and **no keyword** as so far. It can perform attack action.

Reviewer: ```suggestion
Pass a `power` and `toughness` to the `Creature` class:
```

What you had written is perhaps more than necessary here. Just a simple instruction is all that is needed.

```python
>>> creature = ertai.Creature(title="Selfless Savior",cost=ertai.Mana("White"),

> ...            tapped=False,

Reviewer: ```suggestion
```

I do not believe that line is necessary.

...            power=1,
...            toughness=1)
>>> creature
Selfless Savior    Cost:1 White Mana    Power:1    Toughness:1

```

> ### How to use a creature to attack 

Reviewer: ```suggestion
### How to fight another create
```

Attacking is subtly different than fighting.

```

### How to create a ceature card

Pass a `power` and `toughness` to the `Creature` class:

```python
>>> creature = ertai.Creature(title="Selfless Savior",cost=ertai.Mana("White"),
...            power=1,
...            toughness=1)
>>> creature
Selfless Savior    Cost:1 White Mana    Power:1    Toughness:1

```

> ### How to use a creature to fight 
> 
> We can then use this creature to fight.if **no another creature** set as target in this fight, the `fight` function prints a message says `Need a target`. Otherwise, this fight was sucessfully acted by this creature.
> 
> ```python
> >>> target = ertai.Creature(title="Usher of the Fallen",cost=ertai.Mana("White"),
> ...          power=2,
> ...          toughness=1)
> >>> creature.fight(None)
> Need a target
> 
> ```
> 
> ```python
> >>> creature.fight(target)
> >>> creature.is_alive
> False

Reviewer: ````suggestion
### How to use a creature to fight 

We can then use this creature to fight a `target` create:

```python
>>> target = ertai.Creature(title="Usher of the Fallen",cost=ertai.Mana("White"),
...          power=2,
...          toughness=1)
>>> creature.fight(target)
```

In this case the create is no longer alive after the fight:

```python
>>> creature.is_alive()
False
````
